### PR TITLE
Finding last digit of nth fibonacci number

### LINKED ISSUE #1694: Last Digit of nth Fibonacci number by c++
I will give optimised code for finding last digit of nth fibonacci number

### PR DESCRIPTION
**Pull request by RedEye00**
Fixes #1694 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Has to be Reviewed

#### Languages Used:
- C++

#### Files Added:
- Last_Digit_of_Nth_Fibonacci_Number.cpp

Thanks!

### CODE REVIEW

**Comment by RedEye00** [comment]
It fixes issue #1694 sorry I forgot to mention

**Comment by singh-shreya6** [comment]
Please edit the description and add issue number after Fixes #

File: Last_Digit_of_Nth_Fibonacci_Number/Last_Digit_of_Nth_Fibonacci_Number.cpp [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+int main(){
+    int a[60], n, i;
+    a[0] = 0;
+    a[1] = 1;
+    //Using the concept that after 60 numbers in a fibonacci numbers the last digit repeats itself again and again
+    for(i = 2 ; i < 60 ; i++ ){        //for storing the last digits of first 60 fibonacci numbers
+        a[i] = (a[i-1] + a[i-2]) % 10;//Now we will store last digits in array and their index as position in first 60 numbers fibonacci numbers 
+    } 
+    cin>> n ;
```
Comment (singh-shreya6) [inline Last_Digit_of_Nth_Fibonacci_Number/Last_Digit_of_Nth_Fibonacci_Number.cpp]: space before >>

File: Last_Digit_of_Nth_Fibonacci_Number/Last_Digit_of_Nth_Fibonacci_Number.cpp [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    a[1] = 1;
+    //Using the concept that after 60 numbers in a fibonacci numbers the last digit repeats itself again and again
+    for(i = 2 ; i < 60 ; i++ ){        //for storing the last digits of first 60 fibonacci numbers
+        a[i] = (a[i-1] + a[i-2]) % 10;//Now we will store last digits in array and their index as position in first 60 numbers fibonacci numbers 
+    } 
+    cin>> n ;  
+
+    cout<< "\nThe Position of Fibonacci Number: " << n ; //getting the required number
+
+    cout<< "\nThe Last Digit of this Number : " << a[n % 60] ;//print the last digit given in that position
```
Comment (singh-shreya6) [inline Last_Digit_of_Nth_Fibonacci_Number/Last_Digit_of_Nth_Fibonacci_Number.cpp]: space before <<

File: Last_Digit_of_Nth_Fibonacci_Number/Last_Digit_of_Nth_Fibonacci_Number.cpp [hunk 3]
```diff
@@ -0,0 +1,31 @@
+//C++ Program to find last digit of nth fibonacci numbers
+#include<iostream>
+using namespace std;
+
+int main(){
```
Comment (singh-shreya6) [inline Last_Digit_of_Nth_Fibonacci_Number/Last_Digit_of_Nth_Fibonacci_Number.cpp]: space before {

**Comment by RedEye00** [comment]
sorry for delay

**Comment by RedEye00** [comment]
I am a GSOC participant so can i make new issues

**Pull request merged**